x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/kibana_client.ts

```diff
@@ -86,13 +86,14 @@ export interface ChatClient {
 export class KibanaClient {
   axios: AxiosInstance;
   constructor(
-    private readonly log: ToolingLog,
+    public readonly log: ToolingLog,
```

Comment:
you can import `logger` from `services` if you need it, so you shouldn't have to make this public: https://github.com/elastic/kibana/blob/a0f5a7f3fc23d4ba241bc511ddd2987a679ec00d/x-pack/platform/plugins/shared/inference/scripts/evaluation/services/index.ts#L35